ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -935,12 +958,14 @@ static PrjFS_Result HandleRecursivelyEnumerateDirectoryRequest(const MessageHead
         PrjFS_Result result = HandleEnumerateDirectoryRequest(request, path, directoryRelativePath.c_str());
         if (result != PrjFS_Result_Success)
         {
+            LogWarning("HandleRecursivelyEnumerateDirectoryRequest: HandleEnumerateDirectoryRequest failed on %s", path);
             goto CleanupAndReturn;
         }
         
         DIR* directory = opendir(path);
         if (nullptr == directory)
         {
+            LogWarning("HandleRecursivelyEnumerateDirectoryRequest: failed to open dir %s", path);
```

Comment:
It might help to log the specific error (`errno`/`strerror(errno)`) here and in a few of the other new logging statements.